### PR TITLE
[WIP] Nek5000 BP5

### PR DESCRIPTION
**Pull request by thilinarmtb**
Still a work in progress.

- Needs refactoring and cleaning the git history

### CODE REVIEW

**Comment by jeremylt** [comment]
FYSA, the active-passive merge will be soon, which will affect this branch

**Comment by thilinarmtb** [comment]
@jeremylt; Thanks for the headsup ! Will refactor accordingly.

**Comment by jeremylt** [comment]
Active Passive is now merged. The Nek BP1 and BP3 are a bit funny right now, so let me know if you want to work on them together.

File: examples/nek5000/make-nek-examples.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@
 #CC=
 
 ## list of examples to make
-EXAMPLES=(bp1 bp3)
+EXAMPLES=(bp5)
 
 ###############################################################################
 # DONT'T TOUCH WHAT FOLLOWS !!!
```
Comment (jedbrown) [inline examples/nek5000/make-nek-examples.sh]: I guess this comment is asking for it. ;-)

File: examples/nek5000/make-nek-examples.sh [hunk 2]
```diff
@@ -32,7 +32,7 @@
 #CC=
 
 ## list of examples to make
-EXAMPLES=(bp1 bp3)
+EXAMPLES=(bp5)
```
Comment (jedbrown) [inline examples/nek5000/make-nek-examples.sh]: I assume this is temporary and will be fixed in a subsequent commit?

File: examples/nek5000/run-nek-example.sh [hunk 3]
```diff
@@ -30,7 +30,7 @@ fi
 
 # Set defaults for the parameters
 : ${CEED_DIR:=`cd ../../; pwd`}
-nek_examples=(bp1 bp3)
+nek_examples=(bp5)
```
Comment (jedbrown) [inline examples/nek5000/run-nek-example.sh]: Same here?

Comment (thilinarmtb) [reply examples/nek5000/run-nek-example.sh]: Yes, it should be changed. Will do.

**Comment by thilinarmtb** [comment]
I did not get a chance to debug what is going on here. May be @jeremylt  and I can work on it.

**Comment by jeremylt** [comment]
I have a PR to streamline how we handle identity restrictions that should help with some of this.

**Pull request closed without merging**